src/test/resources/org/jabref/logic/importer/fileformat/EndnoteXmlImporterTestLabelAndMultiTitle.bib

```diff
@@ -15,6 +16,7 @@ @article{
 }
 @book{,
       address = {Univ Chicago, Chicago, IL USA},
+      alt-title = {Ecology},
```

Comment:
(comment for below)

Here, I see `journal` set. This is wrong. If the entry is not an `article`, that field should go into `booktitle` or another field. Please try to check the table https://en.wikibooks.org/wiki/LaTeX/Bibliography_Management#BibTeX and think on a case-by-case basis. As fallback, just use `booktitle` before loosing the field.

Reply:
In the original import test xml, it was present as ```alt-title```.
Somewhere I had read during all this that ```booktitle``` in the bib is handled by ```secondary-title``` in the Endnote xml format. So I'll modify the bib and the xml, and change the StandardField of secondary-type to BOOKTITLE.

Reply:
Nope, have to undo this. It's importing as booktitle even for articles. Will try to do it the way you suggested.